Source/WebCore/css/StyleProperties.cpp

```diff
@@ -1242,6 +1243,67 @@ String StyleProperties::borderImagePropertyValue(const StylePropertyShorthand& s
     return result.toString();
 }
 
+String StyleProperties::borderRadiusShorthandValue(const StylePropertyShorthand& shorthand) const
+{
+    ASSERT(shorthand.length() == 4);
+    RefPtr<CSSPrimitiveValue> horizontalRadii[4];
+    RefPtr<CSSPrimitiveValue> verticalRadii[4];
+    CSSValueID cssWideKeyword = CSSValueInvalid;
+    for (unsigned i = 0; i < 4; ++i) {
+        auto value = getPropertyCSSValue(shorthand.properties()[i]);
+        ASSERT(!value || is<CSSPrimitiveValue>(*value));
+        if (!value || !is<CSSPrimitiveValue>(*value))
+            return String();
+
+        // FIXME: Remove this isCSSWideKeyword check after we do this consistently for all shorthands in getPropertyValue.
+        if (CSSValueID id = downcast<CSSPrimitiveValue>(*value).valueID(); isCSSWideKeyword(id)) {
```

Comment:
`auto`?

Also since I prefer words to abbreviations I might name the local `keyword`.